.github/workflows/build-test-publish-images.yml

```
@@ -267,6 +267,37 @@ jobs:
           GPUCIBOT_DOCKERHUB_TOKEN: ${{ secrets.GPUCIBOT_DOCKERHUB_TOKEN }}
           ARCHES: ${{ toJSON(matrix.ARCHES) }}
         run: ci/create-cuvs-multiarch-manifest.sh
+  validate:
+    needs: [compute-matrix, build-rapids, build-rapids-multiarch-manifest]
+    strategy:
+      matrix: ${{ fromJSON(needs.compute-matrix.outputs.TEST_MATRIX) }}
+      fail-fast: false
+    secrets: inherit
+    uses: ./.github/workflows/validate.yml
+    with:
+      ARCH: ${{ matrix.ARCH }}
+      CONTAINER_CANARY_VERSION: 42e3d1186c7750a863321adebeedc2e607e0992a # latest main
```

```suggestion
      CONTAINER_CANARY_VERSION: 'v0.5.0'
```

Latest `main` was just released as `v0.5.0`: https://github.com/NVIDIA/container-canary/releases/tag/v0.5.0